src/swift/Queue.swift

```diff
@@ -12,6 +12,8 @@
 
 // dispatch/queue.h
 
+import CDispatch
```

Comment:
Does this suffer from the same failure as CoreFoundation in swift-corelibs-foundation where we inadvertently expose those methods to apps?